pkg/oc/cli/admin/release/coreos.go

```diff
@@ -0,0 +1,105 @@
+package release
+
+// This package parses the HTTP API effectively
+// created by https://github.com/coreos/coreos-assembler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// BuildMeta is a partial deserialization of the `meta.json` generated
+// by coreos-assembler for a build.
+type BuildMeta struct {
+	AMIs []struct {
+		HVM  string `json:"hvm"`
+		Name string `json:"name"`
+	} `json:"amis"`
+	BuildID string `json:"buildid"`
+	Images struct {
```

Comment:
I'd just drop this and include only the keys we need for now. Not that we're planning to break the schema, though there's no point in deserializing it if we're not using it.

Speaking of, I think we discussed adding schema versioning to the JSON output. Probably would be good to do this before teaching the origin about it?